packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -199,6 +199,7 @@ const Breakouts = WebexPlugin.extend({
    */
   updateBreakout(params) {
     this.set(params);
+    this.set('groups', params.groups);
```

Comment:
imo shouldn't this be a constant?

Reply:
The groups in params will be existed or not, depends on breakout status